instrumentation/azure-functions/src/main/java/io/opentelemetry/instrumentation/auto/azure/functions/AzureFunctionsInstrumentationHelper.java

```diff
@@ -1,14 +1,13 @@
 package io.opentelemetry.instrumentation.auto.azure.functions;
 
-import com.google.common.base.Strings;
```

Comment:
this is the fix, the other changes are just small cleanups

Reply:
Great :)